api/models/helpers.go

```diff
@@ -373,81 +375,6 @@ func templateHelpers() template.FuncMap {
 
 			return template.HTML(strings.Join(ls, ","))
 		},
-		"entry_task": func(entry ManifestEntry, ps string) template.HTML {
-			mappings := []string{}
-
-			for _, port := range entry.Ports {
-				parts := strings.SplitN(port, ":", 2)
-
-				switch len(parts) {
-				case 1:
-					mappings = append(mappings, fmt.Sprintf(`{ "Fn::Join": [ ":", [ { "Ref": "%sPort%sHost" }, "%s" ] ] }`, UpperName(ps), parts[0], parts[0]))
-				case 2:
-					mappings = append(mappings, fmt.Sprintf(`{ "Fn::Join": [ ":", [ { "Ref": "%sPort%sHost" }, "%s" ] ] }`, UpperName(ps), parts[0], parts[1]))
-				}
-			}
-
-			envs := make([]string, 0)
-			envs = append(envs, fmt.Sprintf("\"PROCESS\": \"%s\"", ps))
-
-			for _, env := range entry.Env {
-				parts := strings.SplitN(env, "=", 2)
-				if len(parts) == 2 {
-					envs = append(envs, fmt.Sprintf("\"%s\": \"%s\"", parts[0], parts[1]))
-				}
-			}
-
-			links := make([]string, len(entry.Links))
-
-			for i, link := range entry.Links {
-				name, _, err := linkParts(link)
-
-				if err != nil {
-					continue
-				}
-
-				// Don't define any links for now, as they won't work with one TaskDefinition per process
-				links[i] = fmt.Sprintf(`{ "Fn::If": [ "Blank%sService", { "Ref" : "AWS::NoValue" }, { "Ref" : "AWS::NoValue" } ] }`, UpperName(name))
-			}
-
-			services := make([]string, len(entry.Links))
-
-			for i, link := range entry.Links {
-				name, _, err := linkParts(link)
-
-				if err != nil {
-					continue
-				}
-
-				services[i] = fmt.Sprintf(`{ "Fn::If": [ "Blank%sService", { "Ref" : "AWS::NoValue" }, { "Fn::Join": [ ":", [ { "Ref" : "%sService" }, "%s" ] ] } ] }`, UpperName(name), UpperName(name), name)
-			}
-
-			volumes := []string{}
-
-			for _, volume := range entry.Volumes {
-				if strings.HasPrefix(volume, "/var/run/docker.sock") {
-					volumes = append(volumes, fmt.Sprintf(`"%s"`, volume))
-				}
-			}
-
-			l := fmt.Sprintf(`{ "Fn::If": [ "Blank%sService",
-			{
-				"Name": "%s",
-				"Image": { "Ref": "%sImage" },
-				"Command": { "Ref": "%sCommand" },
-				"Memory": { "Ref": "%sMemory" },
-				"Environment": {
-					"KINESIS": { "Ref": "Kinesis" },
-					%s
-				},
-				"Links": [ %s ],
-				"Volumes": [ %s ],
-				"Services": [ %s ],
-				"PortMappings": [ %s ]
-			}, { "Ref" : "AWS::NoValue" } ] }`, UpperName(ps), ps, UpperName(ps), UpperName(ps), UpperName(ps), strings.Join(envs, ","), strings.Join(links, ","), strings.Join(volumes, ","), strings.Join(services, ","), strings.Join(mappings, ","))
-
-			return template.HTML(l)
-		},
 		"ingress": func(m Manifest) template.HTML {
```

Comment:
Can we get rid of other helpers too? A quick check shows that "ingress" is dead code too.